### ISSUE TITLE
Payment Page Processing Issues

### ISSUE THREAD

**Issue by Mishul-G** [brainblocks/brainblocks#11]
Hey, first wanted to say a big thank you for designing this plugin. This will be key in adoption of xrb and actually bringing value to this coin. 

I wanted to report a bug my website is having - our **xrb payment button** keeps randomizing wallets when we try to pay. We have installed the latest plugin you posted.

In debugging measures, we have also tried to hardcode our xrb-wallet directly into the plugin code, but this either causes the button to not render or not portray any changes. Note - the address that shows up is random everytime we press pay with xrb.

We are designing an XRB marketplace, you can checkout this problem here - http://blockverse.io/index.php/shop/

Just follow through with adding a cart and on the checkout page you will notice this problem in the pay with xrb button. Would really appreciate your help in this. Thank you.

Best,
Mishul Gupta

**Comment by bluepnume** [comment]
Hi @Mishul-G -- that's intentional. We generate a unique account address for each buyer. That way, we can identify who has sent each payment, and we can also automatically refund buyers who accidentally pay more raiblocks than they intended to.

Please feel free to try this out -- set up a a button with your address, for 0.001 XRB, and try sending a payment to it. You'll see that after the payment lands in the unique account, it will be automatically sent from there to the address you configured in the button.

Hope this helps!

**Comment by Mishul-G** [comment]
Hi,

That's what I thought so that the transactions are being processed through
your wallets. We have tried sending XRB to the wallets after we set *our
wallet *as the destination. The transaction was pending overnight and still
has not gone through so I cancelled it. What would you suggest? I am going
to try paying again.

Thanks for the swift response!

Best,
Mishul Gupta

**Comment by Mishul-G** [comment]
Now worries! We got our payment system working. Thanks for setting up this
great plugin - we at BlockVerse (an XRB Marketplace) really believe in XRB.

We have our alpha release coming next week, and we have already partnered
up with several vendors and resorts to provide products and services using
XRB. We have full support for Brainblocks, and hope you guys will help us
back! Here's our pre-alpha website: blockverse.io

Will reach out to you guys during alpha release. Again, great job on the
plugin. Thank you.

Best,
Mishul Gupta
CEO of BlockVerse
